DB/pgsql.php

```diff
@@ -1119,4 +1159,3 @@ function _checkManip($query)
  * End:
  */
 
-?>
```

Comment:
Did you mean to remove this line?  This file should be consistent with the rest of the library.